egs/aishell/v1/run.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Copyright 2017 Beijing Shell Shell Tech. Co. Ltd. (Authors: Hui Bu)
+#           2017 Jiayu Du
+#           2017 Chao Li
+#           2017 Xingyu Na
+#           2017 Bengu Wu
+#           2017 Hao Zheng
+# Apache 2.0
+
+# This is a shell script, but it's recommended that you run the commands one by
+# one by copying and pasting into the shell.
+# Caution: some of the graph creation steps use quite a bit of memory, so you
+# should run this on a machine that has sufficient memory.
```

Comment:
The comment about graph creation is out of date. You might want to have another comment here that reads something like:

`# See README.txt for more info on data required.`
`# Results (mostly EERs) are inline in comments below.`

Reply:
And, please add a README.txt if there isn't one already.